mevislab.github.io/content/about/about.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Overview"
+date: 2022-06-15T08:54:53+02:00
+draft: false
+status: "OK"
+weight: 200
+tags: ["Symbols", "Glossary", "Overview"]
+menu: 
+  main:
+    identifier: "about"
+    title: "Overview"
+    weight: 200
+---
+## Symbols
+We embedded three symbols, referencing additional info, tasks and warnings:
+{{<alert class="info" caption="Info">}}
+Provides additional links or info on the current topic.
+{{</alert>}}
+{{<alert class="check" caption="Check">}}
+Points out a related task.
+{{</alert>}}
+{{<alert class="warning" caption="Warning">}}
+Hints common mistakes or steps you should consider in beforehand.
```

Comment:
in beforehand -> beforehand

Reply:
Will change that immediately

Reply:
I pressed the wrong button, this should have been part of the larger review...